source/includes/_samples.md

```diff
@@ -0,0 +1,66 @@
+# Sample Code
+
+## Resizing an `iframe` Based on Content Changes
+
+> iframe Code:
+
+```javascript
+document.body.style.overflow = 'hidden';
+
+function sendResizeMessage() {
+  window.parent.postMessage({
+    type: 'IFRAME_HEIGHT_RESIZE',
+    frameHeight: document.body.offsetHeight + 10 /* a little extra for good measure */
+  }, '*');
+}
+
+if (window.ResizeObserver) {
+  var heightObserver = new ResizeObserver(function() {
+    sendResizeMessage();
+  });
+  heightObserver.observe(document.body);
+} else {
+  // IE 11
+  setInterval(
+    sendResizeMessage,
+    500 /* ms */
+  );
+}
+```
+
+> Integration Code:
+
+```javascript
+(function (WIAPI) {
+  API = new WIAPI();
+
+  API.insert('content', function (elem, meta) {
+    var iframeElem = document.createElement('iframe');
+    iframeElem.src = '/path-to-iframe.htm';
+    iframeElem.classList.add('my-integration-name-iframe');
+    API.append(elem, iframeElem);
+  });
+
+  function setIframeHeight(e) {
+    if (e.origin !== 'https://yourdomain.com') {
```

Comment:
Just adding `www.` to match the `iframeElem.src` example above, plus most domains use the `www.` subdomain.

```suggestion
    if (e.origin !== 'https://www.yourdomain.com') {
```